doc/source/skips/1-values.rst

```diff
@@ -0,0 +1,168 @@
+======================================
+SKIP-1: scikit-image mission statement
+======================================
+
+:Author: Juan Nunez-Iglesias <juan.nunez-iglesias@monash.edu>
+:Status: Active
+:Type: Process
+:Created: 2018-12-08
+:Resolved:
+:Resolution:
+:Version effective: 0.16
+
+Abstract
+--------
+
+scikit-image should adopt the document below as its mission statement. This
+statement will feature prominently in the scikit-image home page and readme,
+as well as the contributor and core developer guides. Decisions about the API
+and the future of the library would be referenced against this document. (See
+:ref:`governance`.)
+
+In July 2018, I published a blog post that broadly outlined what I would want
+from a roadmap for scikit-image [1]_, but requested comments from the community
+before it would be finalized. I consider that we have collected comments for
+long enough and can move forward with formal adoption. Most comments were
+positive, so below I'll just summarize the “negative” comments under “rejected
+ideas”.
+
+Detailed description
+--------------------
+
+(Or: What problem does this proposal solve?)
+
+Over the past few years, scikit-image has been slightly “adrift”, with new and
+old contributors coming in, adding what small bits they need at the time, and
+disappearing again for a while. This is *fine* and we want to encourage more of
+it, but it also lacks direction. Additionally, without a concerted roadmap to
+concentrate effort, many of these contributions just fall by the wayside, as it
+is difficult for new contributors to live up to our stringent (and largely
+unwritten) standards of code.
+
+Implementation
+--------------
+
+Our mission
+***********
+
+scikit-image aims to be the essential toolkit for scientific image analysis in
+Python. We accomplish this by:
+
+- being **easy to use and install**. We are careful in taking on new
+  dependencies, and sometimes cull existing ones, or make them optional.
+- providing a **consistent API**. Conceptually identical arguments have the
+  same name and position in a function signature.
+- **ensuring accuracy**. Test coverage is close to 100% and code is reviewed by
+  at least two core developers before being included in the library.
+- **caring about users’ data**. We have a functional [2]_ API and don't modify
+  input arrays unless explicitly directed to do so.
+- promoting **education** (including self-education) **in image processing**,
```

Comment:
This sentence has too many buzzwords.  Let's simplify and get to the core.

Reply:
I've attempted that, let me know what you think of the updated version.